protobuf/v1/snapshot.proto

```diff
@@ -89,3 +91,19 @@ message NexusCreateSnapshotReplicaStatus {
     string replica_uuid = 1; // uuid of the replica
     uint32 status_code = 2;  // snapshot operation status (0 means successful completion)
 }
+
+/// Request Parameters for Creating Snapshot Clone.
+message CreateSnapshotCloneRequest {
+    string snapshot_uuid = 1; // uuid of the snapshot
+    string clone_name = 2; // snapshot clone name
+    string clone_uuid = 3; // uuid of the clone to be created from snapshot
+}
+
+/// Request Parameters for List Snapshot Clone.
+message ListSnapshotCloneRequest {
+    optional string snapshot_uuid = 1; // if snapshot uuid is present, return snapshot specific clone, else return all clones
```

Comment:
nitpick:

`if snapshot uuid is present, return clones created from this id, else return all clones available.`